flowmachine/flowmachine/utils/utils.py

```diff
@@ -240,6 +242,54 @@ def proj4string(conn, crs=None):
     return proj4_string.strip()
 
 
+def verify_columns_exist_in_all_tables(conn, tables, columns):
+    """
+    Parse a list of tables ensuring that certain columns are present.
+
+    Parameters
+    ----------
+    conn : Connection
+        FlowMachine db connection to use to get the subscriber_tables.
+    tables : str or list of strings, default 'all'
+        Can be a sting of a single table (with the schema) or a list of these.
+        The keyword all is to select all subscriber tables
+    columns : str or list
+        A string or list of strings with the column names that must be present.
+
+    Returns
```

Comment:
I would suggest to not return anything from this function, because the return value will be exactly equal to the input `tables` (unless one of the columns is missing, in which case an error will be thrown).